spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkBatchQueryScan.java

```diff
@@ -93,6 +93,11 @@
     }
   }
 
+  @Override
+  public String description() {
+    return "IcebergScan " + super.description();
+  }
```

Comment:
Unfortunately the label for the node in the SQL DAG in the Spark UI just says "BatchScan" (that is the `nodeName` for a `BatchScanExec` node, and we can't affect it from Iceberg), but the tool tip for the node contains this `description`.
Also, the detailed Physical Plan shows this `description`. The `toString` method also uses "IcebergScan" (I followed its lead), but I don't see the `toString` being used in the Spark UI.

Reply:
This sounds good to me even outside the scope of this PR, although maybe we sould right Iceberg Batch Scan, since we will probably be generating different descriptions for SparkFileScan and any other Scans we build

Reply:
Ah I see we have the other descriptions below, well at least let's match that pattern :) "IcebergBatchScan"?

Reply:
I was matching the pattern in the `toString` method for each of the subclasses of `SparkScan`. If you prefer, we can use "IcebergBatchScan" in both `description` and `toString` for `SparkBatchQueryScan`.

Reply:
I am +1 on `BatchScan` -> `IcebergBatchScan` for UI purposes.

I think that updating the `toString` method should be fine as well if need be, provided it passes tests etc. `IcebergBatchScan` is closer to the name of the class anyway than plain `IcebergScan`.